README.md

```diff
@@ -34,3 +34,7 @@ The locked versions for deployment are in `requirements.txt`.
 To update the locked versions:
 
     $ sudo docker-compose run web bash -c 'pip freeze > requirements.txt'
+
+## License
+
+The Montréal-Python is Apache licensed.
```

Comment:
Oops forgot some words: The Montréal-Python website is Apache licensed.  (The blog posts and other text are under CC?)